### ISSUE TITLE
Problem in documentation - Update method

### ISSUE THREAD

**Issue by LaurentioVerde** [hajimehoshi/ebiten#1402]
### Summary
I've encountered problem in documentation. When I was trying to copy and run code from:
https://ebiten.org/tour/hello_world.html
I've got this error:
```
./main.go:27:27: cannot use &Game literal (type *Game) as type ebiten.Game in argument to ebiten.RunGame:
        *Game does not implement ebiten.Game (wrong type for Update method)
                have Update(*ebiten.Image) error
                want Update() error
```

### How to fix
Change code from documentation:
```
func (g *Game) Draw(screen *ebiten.Image) {
	ebitenutil.DebugPrint(screen, "Hello, World!")
}
```
to:
```
func (g *Game) Draw() {
	ebitenutil.DebugPrint(screen, "Hello, World!")
}
```

**Comment by hajimehoshi** [comment]
The documentation is about v1, not v2. v2.0.0 is not released yet.

**Comment by lathompson** [comment]
I think the 2.0 alpha is now on the master branch. Was that intentional?

**Comment by hajimehoshi** [comment]
Yes. You mean v2.1.0-alpha? There is also 2.0 branch and v2.0.0-rc.2 tag.

**Comment by hajimehoshi** [comment]
So as v2.0.0 is not released yet, 2.0 is not available unless you specify the tag explicitly. (I assume you use Go modules).

I'll update the document at the same time when v2.0.0 is released. Stay tuned!
